hybrasyl/Objects/User.cs

```diff
@@ -2305,586 +2145,103 @@ public override void RegenerateMp(double mp, Creature regenerator = null)
-                                        }
-                                        break;
-                                }
-                            }
-                            break;
-                        case IntentDirection.Right:
-                            {
-                                switch (Direction)
-                                {
-                                    case Direction.North:
-                                        {
-                                            //facing north, attack east
-                                            rect = new Rectangle(this.X, this.Y, 1 + intent.Radius, 1);
-                                        }
-                                        break;
-                                    case Direction.South:
-                                        {
-                                            //facing south, attack west
-                                            rect = new Rectangle(this.X - intent.Radius, this.Y, intent.Radius, 1);
-                                        }
-                                        break;
-                                    case Direction.East:
-                                        {
-                                            //facing east, attack south
-                                            rect = new Rectangle(this.X, this.Y - intent.Radius, 1, intent.Radius);
-                                        }
-                                        break;
-                                    case Direction.West:
-                                        {
-                                            //facing west, attack north
-                                            rect = new Rectangle(this.X, this.Y, 1, 1 + intent.Radius);
-                                        }
-                                        break;
-                                }
-                            }
-                            break;
-                        case IntentDirection.Nearby:
-                            {
-                                //attack radius
-                                rect = new Rectangle(this.X - intent.Radius, this.Y - intent.Radius, intent.Radius * 2, intent.Radius * 2);
-                            }
-                            break;
-                    }
-
-                    if (!rect.IsEmpty) possibleTargets.AddRange(Map.EntityTree.GetObjects(rect).Where(obj => obj is Creature && obj != this && obj.GetType() != typeof(Merchant))); ;
-
-                    var actualTargets = intent.MaxTargets > 0 ? possibleTargets.Take(intent.MaxTargets).OfType<Creature>().ToList() : possibleTargets.OfType<Creature>().ToList();
-
-                    foreach (var target in actualTargets)
+                    Status applyStatus;
+                    if (World.WorldData.TryGetValue<Status>(status, out applyStatus))

```

Comment:
applyStatus should be declared inline. https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-7

Reply:
Huh. Neat! Will fix.